xmtp_mls/src/client.rs

```diff
@@ -607,18 +607,21 @@ where
                     // provider alone.
                     let provider_ref = &provider;
                     async move {
+                        let mls_group = group.load_mls_group(provider_ref)?;
                         log::info!("[{}] syncing group", self.inbox_id());
                         log::info!(
                             "current epoch for [{}] in sync_all_groups() is Epoch: [{}]",
                             self.inbox_id(),
-                            group.load_mls_group(provider_ref)?.epoch()
+                            mls_group.epoch()
                         );
+                        if mls_group.is_active() {
```

Comment:
This is going to have merge conflicts with https://github.com/xmtp/libxmtp/pull/990

FYI @insipx